src/main/java/gregtech/api/util/GTPP_Recipe.java

```diff
@@ -486,7 +486,7 @@ protected void drawNEIOverlayForInput(FixedPositionedStack stack) {
                 "",
                 4,
                 4,
-                0,
+                1,
```

Comment:
Actually chemplant references `sChemicalPlant_GT` which is a copy of `sChemicalPlantRecipes`. That's why it doesn't work well. Can you remove the former and use the latter instead?

Reply:
Also: you changed `aMinimalInputItems`, not `aUsualInputCount`. Please revert it